.github/workflows/check_version.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
       - name: Compare versions
         run: |
           SNAP_VERSION=$(yq .version snap/snapcraft.yaml)
+          sudo apt-get update
           APT_VERSION=$(apt-cache show mysql-server-8.0 | grep Version | head -n 1 | cut -c10-)
```

Comment:
IMHO: shorter with the same result:
```
apt-cache show mysql-server-8.0 | awk '/Version: /{print $2;exit}'
```

Reply:
addressed in [d05e630](https://github.com/canonical/charmed-mysql-snap/pull/41/commits/d05e63043baf4af61a040e08be847cdfe5f60068)